app/backend/views/devindex.scala.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html ng-app="app">
+<html lang="fi" ng-app="app">
```

Comment:
Mietin et saisko tän asetettua jotenkin dynaamisesti käyttäjän kielivalinnan perusteella? Playn templaateilla ehkä onnistuisi jos vaan käyttäjä on kirjautunut. Tuotannossahan näin on käytännössä aina. 

Mutta ehkei tämän tiketin asiaa :)

Reply:
Tällä kieliattribuutilla saattaa muuten olla vaikutuksia mm. autocorrect ja spell check -asioihin. Muistaisin että joku tiketti oli joskus tähän liittyen, pitäis koittaa kaivaa se esiin niin vois koittaa varmistaa ettei tuo muuta sitä miten se korjaus toimii.

Reply:
Tämä: https://jira.eduuni.fi/browse/CSCEXAM-337

Reply:
Kannattaisikohan tuo siis jättää toistaiseksi vielä laittamatta ja selvittää tuo aikaisempi tiketti ensin?